doc/source/tokudb/tokudb_fractal_tree_indexing.rst

```diff
@@ -0,0 +1,50 @@
+.. _tokudb_fractal_tree_indexing:
+
+=============================
+TokuDB Fractal Tree Indexing
+=============================
+
+Fractal Tree indexing is the technology behind TokuDB and TokuMX, and is
```

Comment:
No need to mention TokuMX, it is long deprecated and EOL

Reply:
removed TokuMX